File: tests/fixtures/python/sample.py
"""Python fixture: 2 top-level fns + 1 class with 2 methods + 1 cross-call."""


def top_level_one(x):
    return helper(x)


def top_level_two(x):
    return x + 1


def helper(x):
    return x * 2


class Greeter:
    def __init__(self, name):
        self.name = name

    def greet(self):
        return helper(len(self.name))

File: tests/fixtures/ruby/sample.rb
# Ruby fixture: top-level methods, class with instance method, cross call.

def helper(x)
  x * 2
end

def top_level_one(x)
  helper(x)
end

class Greeter
  def initialize(name)
    @name = name
  end

  def greet
    helper(@name.length)
  end
end
